domain-services/ifix-es-pipeline/LOCALSETUP.md

```diff
@@ -0,0 +1,18 @@
+# Local Setup
+
+To setup the national-dashboard-kafka-pipeline service in your local system, clone the [Core Service repository](https://github.com/egovernments/core-services).
```

Comment:
Name correction on this line and last line

Reply:
minor change, to be taken up as part of code cleanup.